docs/windows.md

```diff
@@ -99,174 +125,80 @@ New-KubeProxyServiceInterface
 
 New-DirectoryIfNotExist "${AntreaHome}/logs"
 New-DirectoryIfNotExist "${KubeProxyLogPath}"
-nssm install kube-proxy "${KubernetesHome}/kube-proxy.exe" "--proxy-mode=userspace --kubeconfig=${KubeProxyKubeconfigPath} --log-dir=${KubeProxyLogPath} --logtostderr=false --alsologtostderr"
+# Install kube-proxy service
+if ($InstallKubeProxy) { nssm install kube-proxy "${KubernetesHome}/kube-proxy.exe" "--proxy-mode=userspace --kubeconfig=${KubeProxyKubeconfigPath} --log-dir=${KubeProxyLogPath} --logtostderr=false --alsologtostderr" }
 nssm install antrea-agent "${AntreaHome}/bin/antrea-agent.exe" "--config=${AntreaHome}/etc/antrea-agent.conf --logtostderr=false --log_dir=${AntreaHome}/logs --alsologtostderr --log_file_max_size=100 --log_file_max_num=4"
 
-nssm set antrea-agent DependOnService kube-proxy ovs-vswitchd
+nssm set antrea-agent DependOnService ovs-vswitchd
+if ($InstallKubeProxy) { nssm set antrea-agent DependOnService kube-proxy ovs-vswitchd }
 nssm set antrea-agent Start SERVICE_DELAYED_AUTO_START
 
-Start-Service kube-proxy
+if ($InstallKubeProxy) { Start-Service kube-proxy }
 Start-Service antrea-agent
 ```
 
 ### Installation as a Pod (Docker/containerd based runtimes)
 
+Starting with Antrea v1.13, Antrea will take over all the responsibilities of kube-proxy for Windows nodes by default.
```

Comment:
Please wrap lines at 80. Do not add a new line between sentences.

Reply:
Updated.